client/src/components/ProtectedRoute.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+const ProtectedRoute = ( { component: Component, ...rest }) => {
+
+    return (
+        <Route 
+            {...rest}
+            render={() => {
+                return localStorage.getItem('token') ? (<Component />) : (<Redirect to="/login" />);
+            }}
+        />
+    )
```

Comment:
With the way this is currently set up it is fine for the scope of this project but typically you'll want to make things as flexible as possible so here you would want to spread rest into Component so that component could be passed props from the component routing it.